ublox/src/ubx_packets/packets/rxm_sfrbx.rs

```diff
@@ -50,3 +69,62 @@ impl core::iter::Iterator for DwrdIter<'_> {
             .map(|bytes| u32::from_le_bytes(bytes.try_into().unwrap()))
     }
 }
+
+#[cfg(feature = "sfrbx-gps")]
+use gnss_protos::{GpsDataByte, GpsQzssDecoder, GpsQzssFrame};
+
+#[derive(Debug, Clone)]
+pub enum RxmSfrbxInterpreted {
+    /// [GpsQzssFrame]
+    #[cfg(feature = "sfrbx-gps")]
+    GpsQzss(GpsQzssFrame),
+}
+
+impl RxmSfrbxRef<'_> {
+    /// Try to interpret the RXM-SFRBX inner frame (when supported/known).
+    pub fn interpret(&self) -> Option<RxmSfrbxInterpreted> {
+        // interpretation is GNSS dependent
+        match self.gnss_id() {
+            0 | 5 => {
+                #[cfg(feature = "sfrbx-gps")]
+                {
+                    let decoded = self.gps_qzss_decoding()?;
+                    Some(RxmSfrbxInterpreted::GpsQzss(decoded))
+                }
+                #[cfg(not(feature = "sfrbx-gps"))]
+                {
+                    None
+                }
+            },
+            _ => {
+                // either not supported or not applicable
+                None
+            },
+        }
+    }
+
+    #[cfg(feature = "sfrbx-gps")]
+    /// [GpsQzssFrame] interpretation attempt, from parsed UBX-SFRBX data.
+    fn gps_qzss_decoding(&self) -> Option<GpsQzssFrame> {
+        let mut decoder = GpsQzssDecoder::default().without_parity_verification();
```

Comment:
The decoder does not allocate anything and is `no-std` compliant.
So It's fine to "build" one per interpretation.
This will apply similarly to other cases (when we unlock them), considering GPS frames are tiny streams of bytes